tests/test_pyb4model.py

```diff
@@ -86,6 +86,11 @@ def test_ForSelect():
 
 
 def test_feature_splitter():
+    """
+Test function for feature splitter
+This function checks if input data if data frame, then splits the data into two parts:
+Which is a tuple containing a list for numeric features and a tupple with numeric features.
+    """
```

Comment:
Could you change the indentation like this:
```suggestion
    """
   Test function for feature splitter
   This function checks if input data if data frame, then splits the data into two parts:
   Which is a tuple containing a list for numeric features and a tupple with numeric features.
    """
```
not sure if that will make a difference, just in case